apps/i18n/common/en_us.json

```diff
@@ -19,6 +19,7 @@
   "clearPuzzleConfirmHeader": "Are you sure you want to start over?",
   "codeMode": "Code",
   "codeTooltip": "See generated JavaScript code.",
+  "completedWithoutRecommendedBlock": "Congratulations! You completed Puzzle {puzzleNumber}. (But you could use a different block for stronger code.)",
```

Comment:
I agree that those sound nicer. I'm not sure I'm convinced that they're any less judgemental. Of the two, I think I prefer cleaner, although in my mind code cleanliness is somewhat subjective metric.